src/System.Threading.Overlapped/tests/ThreadPoolBoundHandle_IntegrationTests.cs

```diff
@@ -15,7 +16,7 @@ public unsafe void SingleOperationOverSingleHandle()
     {
         const int DATA_SIZE = 2;
 
-        SafeHandle handle = HandleFactory.CreateAsyncFileHandleForWrite(@"SingleOverlappedOverSingleHandle.tmp");
+        SafeHandle handle = HandleFactory.CreateAsyncFileHandleForWrite(Path.Combine(TestDirectory, @"SingleOverlappedOverSingleHandle.tmp"));
```

Comment:
I'm so sorry I merged and forgot to address this last issue. I'll end out a follow up PR to fix it.